dotCMS/src/main/java/com/dotmarketing/business/ExpiredContentletPublishStateException.java

```diff
@@ -0,0 +1,17 @@
+package com.dotmarketing.business;
+
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+
+public class ExpiredContentletPublishStateException extends PublishStateException {
+    private Contentlet contentlet;
+
+    public ExpiredContentletPublishStateException(final Contentlet contentlet) {
+        super("\"The content cannot be published because the expire date has already passed.\"");
```

Comment:
remove the double quotes such as
```
super ("The content cannot be published because the expire date has already passed.")
```

Reply:
Also, add a property that would allow users/devs to identify what specific Contentlet was not published.

Reply:
this property already exists

https://github.com/dotCMS/core/pull/19988/files/f79d75c145af00145cb048876e2203676ea9ebff#diff-bec1c25354d60d340e39396f824da23ba8de789736846051e3dad19dfb2a82a8R6

Reply:
done https://github.com/dotCMS/core/pull/19988/commits/30658f7295ec9788fbc12fcd61c2f3b3bf07e272